idea-plugin/src/main/java/com/palantir/javaformat/intellij/PalantirJavaFormatSettings.java

```diff
@@ -91,6 +91,20 @@ Optional<List<URI>> getImplementationClassPath() {
         return state.implementationClassPath;
     }
 
+    /**
+     * The path to the formatter nativeImage.
+     */
+    Optional<URI> getNativeImageClassPath() {
+        return state.nativeImageClassPath;
+    }
+
+    /**
+     * Whether to use the legacy implementation(getImplementationClassPath) or the new implementation(getNativeImageClassPath)
+     */
+    Boolean getUseNativeImageClassPath() {
```

Comment:
Does this handle the case where this config is not here at all? I think it's probably best to support the `useNativeImageClassPath` not existing for two reasons:

1. People could update the plugin then it might crash if the user opens the settings page on a project that hasn't resynced gradle yet and so doesn't have the config - I think I saw this yesterday.
2. If we ever want to remove it later, should make it optional so when people upgrade the gradle plugin before the intellij plugin it still works.

Reply:
Actually, can we just use whether or not `getNativeImageClassPath` exists to determine whether or not to use the native formatter? And just remove this boolean entirely?